Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:16.04
```

Comment:
total curiosity here - could we base this off of our ubuntu slim-based [docker-base img](https://github.com/deis/docker-base)? for our purposes here, we'd really only get `curl` and `bash`... but maybe it'd reduce img size?